pyiron/atomistics/structure/pyironase.py

```diff
@@ -22,9 +14,6 @@
 __status__ = "production"
 __date__ = "Sep 1, 2017"
 
-sys.modules["ase.atom"] = atom
```

Comment:
If you remove these lines the overwriting of ASE atoms does not work any more. Namely when you use ASE functions they are going to return ASE objects, rather than pyiron objects. While this fixes the issue with the write function, it is a very dangerous change other users might rely on.